apps/nextjs-website/src/lib/fetch/types/page.ts

```diff
@@ -0,0 +1,127 @@
+import * as t from 'io-ts';
+
+const HeroSectionCodec = t.strict({
+  __component: t.literal('sections.hero'),
+  title: t.string,
+  subtitle: t.union([t.string, t.null]),
+  theme: t.union([t.literal('light'), t.literal('dark')]),
+  inverse: t.boolean,
+  size: t.keyof({
+    small: null,
+    big: null,
+  }),
+  useHoverlay: t.boolean,
+  sectionID: t.union([t.string, t.null]),
+  image: t.union([
+    t.strict({
+      alternativeText: t.union([t.string, t.null]),
+      url: t.string,
+    }),
+    t.null,
+  ]),
+  background: t.union([
+    t.strict({
+      alternativeText: t.union([t.string, t.null]),
+      url: t.string,
+    }),
+    t.null,
+  ]),
+  ctaButtons: t.array(
+    // TODO: Replace with CTAButtonSchema when merged
```

Comment:
Is this a typo? The `CTAButtonSchema` is available. I'd do it before the merge

Reply:
Not a typo.

The comment was just created when CTAButtonSchema wasn't merged into main yet.

I updated the ctaButtons field and the image/background field (which was in the same situation, just for StrapiImageSchema)